app/utils/IncidentP5Helper.scala

```diff
@@ -163,6 +163,32 @@ class IncidentP5Helper(
     ).flatten
   )
 
+  def containerIdentificationNumberRow(equipmentIndex: Index): Option[SummaryListRow] =
+    buildRowFromAnswer[String](
+      answer = Some(data.TransportEquipment(equipmentIndex.position).containerIdentificationNumber).flatten,
+      formatAnswer = formatAsText,
+      prefix = "departure.notification.incident.index.transportEquipment.containerIdentificationNumber",
+      id = None,
+      call = None
+    )
+
+  def transportEquipmentSection(equipmentIndex: Index): AccordionSection =
+    AccordionSection(
+      sectionTitle = Some(messages("departure.notification.incident.index.transportEquipment.heading", equipmentIndex.display)),
+      rows = Seq(containerIdentificationNumberRow(equipmentIndex)).flatten,
+      isOpen = true
```

Comment:
Presumably only if the first index should this be true?

Reply:
Potentially, maybe I've misinterpreted it:
"By default, the first accordion to remain open and to display Container identification number. All sub accordions to remain closed."
Each transport equipment accordion has a container id number so I thought they would all be open but the children accordions (seals/goods ref) would be closed. Have I got the wrong end of the stick?